playbooks/roles/edxapp/tasks/deploy.yml

```diff
@@ -423,3 +423,14 @@
   tags:
     - manage
     - deploy:platform
+
+- name: Create OAuth credentials
+  shell: >
+    {{ edxapp_venv_bin }}/python manage.py lms --setting=aws create_oauth2_client {{ item.url }} {{ item.redirect_uri }}
```

Comment:
@johnbaldwin Good point. I _think_ that it doesn't actually matter in this case since we're just adding some tokens to the database. The extra items in [our settings](https://github.com/appsembler/edx-platform/blob/appsembler/master/cms/envs/aws_appsembler.py) shouldn't have an effect on this task, it would be better to use `aws` if possible so we can push this back upstream. @tkeemon @melvinsoft could you chime in on this?